### PR TITLE
snort 2.9.16

### PR DESCRIPTION
**Pull request by bayandin**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by bayandin** [comment]
Currently, it fails with an error:

```
==> brew install ./snort--2.9.16.catalina.bottle.tar.gz
Warning: The post-install step did not complete successfully
You can try again using `brew postinstall snort`
==> An exception occurred within a child process:
  Errno::EACCES: Permission denied @ rb_sysopen - /usr/local/etc/snort/attribute_table.dtd
```

**Comment by bayandin** [comment]
Now I see why we need a mirror for `snort`
```
==> Downloading https://www.snort.org/downloads/snort/snort-2.9.16.tar.gz
/usr/bin/curl --disable --globoff --show-error --user-agent Homebrew/2.2.13-34-gb11d2b5\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15.4\)\ curl/7.64.1 --fail --silent --retry 3 --location --remote-time --continue-at 0 --output /Users/brew/Library/Caches/Homebrew/downloads/eb68169074f069f1a9fbae5b655eddffe607a7843f1931f0f3ed668235483806--snort-2.9.16.tar.gz.incomplete https://www.snort.org/downloads/snort/snort-2.9.16.tar.gz
curl: (22) The requested URL returned error: 429 
```

File: Formula/snort.rb [hunk 1]
```diff
@@ -55,7 +54,8 @@ def install
     system "make", "install"
 
     rm Dir[buildpath/"etc/Makefile*"]
-    (etc/"snort").install Dir[buildpath/"etc/*"]
+    (buildpath/"etc").children.each { |f| chmod 0644, f }
```
Comment (SMillerDev) [inline Formula/snort.rb:57]: Not sure what permissions it has but that should probably be reported

Comment (bayandin) [reply Formula/snort.rb:57]: Yup, working on it. Just wanted to check that it helps before reporting (and that's why I've added `do not merge` label)

Comment (bayandin) [reply Formula/snort.rb:57]: Have posted a question to snort-devel@lists.snort.org (as soon as it will be approved by a moderator will add a link)

**Pull request closed without merging**